mmv1/products/vertexai/FeatureGroup.yaml

```diff
@@ -0,0 +1,125 @@
+    name: 'vertex_ai_featuregroup_basic'
+    primary_resource_id: 'test-tf-featuregroup'
+    primary_resource_name: "fmt.Sprintf(\"terraform%s\",
+      context[\"random_suffix\"\
+      ])"
+    vars:
+      name: 'test_tf_featuregroup'
+      project: 'appeng-flex'
+    test_env_vars:
+      org_id: :ORG_ID
+      billing_account: :BILLING_ACCT
+    ignore_read_extra:
+      - 'force_destroy'
+parameters:
+  - !ruby/object:Api::Type::String
+    name: region
+    description: The region of the dataset. eg us-central1
+    url_param_only: true
+    immutable: true
+    default_from_api: true
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    description:
+      The name of the FeatureGroup. This value may be up to 60 characters, and
+      valid characters are [a-z0-9_]. The first character cannot be a number.
+    immutable: true
+    url_param_only: true
+    pattern: projects/{{project}}/locations/{{region}}/featureGroups/{{name}}
+    custom_flatten: templates/terraform/custom_flatten/name_from_self_link.erb
+  - !ruby/object:Api::Type::String
+    name: 'description'
+    description: |
+      description of the FeatureGroup.
+  - !ruby/object:Api::Type::String
+    name: 'etag'
+    description: Used to perform consistent read-modify-write updates.
+    output: true
+    ignore_read: true
+  - !ruby/object:Api::Type::String
+    name: 'createTime'
+    output: true
+    description: |
+      The timestamp of when the featureGroup was created in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+  - !ruby/object:Api::Type::String
+    name: 'updateTime'
+    output: true
+    description: |
+      The timestamp of when the featureGroup was last updated in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: 'labels'
+    description: |
+      A set of key/value label pairs to assign to this featureGroup.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'bigQuery'
+    required: true
+    immutable: true
+    description: |
+       Indicates that features for this group come from BigQuery Table/View. By default treats the source as a sparse time series source, which is required to have an entityId and a feature_timestamp column in the source.
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: 'bigQuerySource'
+        required: true
+        immutable: true
+        description: |
+          The BigQuery source URI that points to either a BigQuery Table or View.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'inputUri'
+            required: true
+            description: |
+              BigQuery URI to a table, up to 2000 characters long. Accepted forms: BigQuery path. For example: bq://projectId.bqDatasetId.bqTableId.
+      - !ruby/object:Api::Type::Array
+        name: 'entityIdColumns'
+        default_from_api: true
```

Comment:
What is the reason to have ` default_from_api` here? Will this field have a default value in API side if it is not provided by the users?